container-builder/util/log/log.go

```diff
@@ -16,119 +16,10 @@
 
 package log
 
-import (
-	"fmt"
-
-	"github.com/go-logr/logr"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
-
-	"github.com/kiegroup/kogito-serverless-operator/container-builder/util"
+// Constants for different log level verbosity.
+const (
```

Comment:
can't you import it from log/log.go?

Reply:
@ricardozanini didn't want to share a module between `kogito-serverless-operator/container-builder` and `kogito-serverless-operator` and preferred to duplicate the constant declarations.